src/provider/FDC3.ts

```diff
@@ -295,8 +295,9 @@ export class FDC3 {
                         .then(() => {
                             clearTimeout(timeout);
                             if (context) {
-                                // Pass context to application before resolving
-                                fin.InterApplicationBus.publish('context', {context}).then(resolve).catch((reason: string) => reject(new Error(reason)));
+                                // Pass context to application before resolving (assume that the main window is the one with listeners registered)
+                                fin.InterApplicationBus.send({...app.identity, name: app.identity.uuid}, 'context', context)
```

Comment:
app.identity will never have a name

This was a quick fix to make it send to the main window of an app since before it was just broadcasting to everyone (which according to the spec it shouldn't). This can (and should) be revisited as part of any provider changes we do.